serverless-schedule-manager/functions/common/helpers/retry-handler.private.js

```diff
@@ -27,27 +27,39 @@ exports.retryHandler = async (error, parameters, callback) => {
   if (!isNumber(parameters.attempts))
     throw new Error('Invalid parameters object passed. Parameters must contain the number of attempts');
 
-  const { TWILIO_SERVICE_MAX_BACKOFF, TWILIO_SERVICE_MIN_BACKOFF, TWILIO_SERVICE_RETRY_LIMIT } = process.env;
+  const { TWILIO_SERVICE_MAX_BACKOFF, TWILIO_SERVICE_MIN_BACKOFF, TWILIO_SERVICE_RETRY_LIMIT, ENABLE_LOCAL_LOGGING } =
```

Comment:
`ENABLE_LOCAL_LOGGING` needs to be added to the env file(s) for serverless-schedule-manager.